onnxruntime/core/framework/allocation_planner.cc

```diff
@@ -1676,7 +1676,7 @@ class PlannerImpl {
 #endif
 
 #if !defined(ORT_MINIMAL_BUILD) && defined(ORT_MEMORY_PROFILE)
-  void CalculateLifetime(const std::vector<int>& ort_value_usecount) {
+  void CalculateLifetime(std::vector<int>& ort_value_usecount) {
```

Comment:
why remove the const?

Reply:
Because ort_value_usecount is changed in line 1704, 1715 and 1728